src/linker/Linker/Message.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Mono.Linker
+{
+	public class Message
+	{
+		public MessageOrigin Origin { get; }
+		
+		public string Subcategory { get; }
+		
+		public MessageCategory Category { get; }
```

Comment:
There is already `MessageImportance` with same purposes and would need unification

Reply:
Thanks. I'll unify it in another PR, to not add much noise in this one.